Code/ThirdParty/dxc/cgmanifest.json

```diff
@@ -5,7 +5,7 @@
         "type": "git",
         "git": {
           "repositoryUrl": "https://github.com/microsoft/DirectXShaderCompiler",
-          "commitHash": "b640fa4ba4afb8eb9f15b4a34ab5187c7d94de97"

```

Comment:
I can't comment on the DXC DLL above.
Is there maybe a better way to ship this? It's quite large and we probably need to update it every once in a while. Maybe we could at least move it into some submodule?

Reply:
I just saw that the .so file is 25 MB, we can't commit that. Can talk offline about that.

Reply:
The windows dll is 18 mb, and that is also checked in, so I didn't think much about it. @Sanakan8472 want's to do a seperate submodule with all the vulkan dependnecies.

Reply:
Yes, but checking that in was already a bad idea. We should fix this now.